src/reference/api/scan.rst

```diff
@@ -0,0 +1,4 @@
+Scan API
+========
+
+.. automodule:: cylc.flow.network.scan_nt
```

Comment:
The bonus to docstring documentation, it's so easy to pull it through to cylc-doc.